### ISSUE TITLE
Day of week across the top doesn't properly align with dates

### ISSUE THREAD

**Issue by andrewoh531** [clauderic/react-infinite-calendar#21]
When the width is quite small as in my case the day of week doesn't align well with the dates due to the scroll bar. Example in the screen shot below 4th of June sits somewhat between Friday and Saturday.

![screen shot 2016-06-19 at 11 30 57 pm](https://cloud.githubusercontent.com/assets/4927570/16177605/2c57cee6-3676-11e6-8b14-7a00b409f424.png)

**Comment by clauderic** [comment]
Well it's either that or have the scrollbar cover the dates. Not sure which is the least offensive...

**Comment by offsky** [comment]
In JS you can inspect the `clientWidth` or `scrollWidth` of the innerScrollContainer. If it is smaller than it's parent's width then you add padding after "Sat" equal to the difference.

**Comment by clauderic** [comment]
Hmm 🤔 Yeah that could work, I'll give it a spin

**Comment by bvaughn** [comment]
You may be aware of this already @clauderic, but react-virtualized uses the `dom-helpers/util/scrollbarSize` helper function for this.

**Comment by clauderic** [comment]
Just published `1.1.15` which fixes this issue

**Comment by offsky** [comment]
I think maybe there is still a small bug with this. The weekday headers now take the scrollbar into account, but the calendar days no longer do so (see screenshot):

![screen shot 2016-06-29 at 11 15 50 am](https://cloud.githubusercontent.com/assets/1110926/16463478/df2a3074-3dea-11e6-8181-15b671683f3c.png)

**Comment by clauderic** [comment]
@offsky That's not what I'm seeing on my end using Chrome on a mac. Can you create a repro of this? Here's a jsfiddle you can fork https://jsfiddle.net/clauderic/4rhn03do/

**Comment by offsky** [comment]
Ok, I may be narrowing in on the issue. I am also using Mac Chrome and the jsfiddle you provided looks fine.  However, my local clone of your repository has the error shown above.  I then built the local repository and ran that, and its ok. 

So the bug that I noticed only appears in the dev environment (`npm start` then http://localhost:3001/).  The build process somehow fixes the problem.

Restoring the List.scss file to the way it was before you made those deletions fixes the problem in the dev environment, but I dont know what the ramification are to the build step if you put in that duplicated code.
